### ISSUE TITLE
Add Binary Thrift over HTTP as a transport option for `OpenTelemetry::Exporters::Jaeger::Exporter`

### ISSUE THREAD

**Issue by konfortes** [open-telemetry/opentelemetry-ruby#231]
Hey guys, 
I'm having some trouble with setting the `OpenTelemetry::Exporters::Jaeger::Exporter` with `opentelemetry-collector`.
This is how I configure it:

```
collector_host = 'otel-collector.opentelemetry.svc.cluster.local'
collector_port = 14268

OpenTelemetry::SDK.configure do |c|
  c.add_span_processor(
    OpenTelemetry::SDK::Trace::Export::SimpleSpanProcessor.new(
      OpenTelemetry::Exporters::Jaeger::Exporter.new(
        service_name: 'ruby-app', host: collector_host, port: collector_port
      )
    )
  )
end
```

it seems like the collector host succesfully resolved because I **DON'T** 
get this error anymore: `/usr/local/bundle/gems/opentelemetry-exporters-jaeger-0.4.0/lib/opentelemetry/exporters/jaeger/transport.rb:14:in** 'connect': getaddrinfo: Name or service not known (SocketError)`

But I can't see any logs related to this ruby-app in the collector nor see any traces in Jaeger.
Am I missing something?

**Comment by konfortes** [comment]
the opentelemetry-collector expects traces in `/api/traces` so I tried the Jaeger collector directly, but the problem persists. keep getting `Unable to connect to Jaeger Agent`

**Comment by fbogsany** [comment]
I won't get to this this week, so punting it to Alpha 0.6. In reality, it should be done well before then, and can be released as a 0.5.1 version of just the Jaeger gem.
